### DISCUSSION TITLE
Install shows it updated but after close opens back to old version

### DISCUSSION THREAD

**Discussion by Doughboy6480** [rgriebl/brickstore#608] [General]
I have tried several times to update to the latest version 2022.9.3 it shows that it updated correctly but when I close the program and reopen I still have 2022.4.1 build 663. Almost to the point of de-installing and reinstalling but not sure that will even help. I'm running Windows 64bit

**Comment by rgriebl** [comment]
The update on 64bit is a bit awkward, because it actually cannot update directly: instead it deinstalls the old BrickStore (from Program Files (x86)) and then installs the new one (to Program Files).
If something is preventing the installer from deinstalling the old one, you'll end up with both. This can happen if the BrickStore is running and won't quit fast enough or if something else in your system is accessing the old installation directory. The problem is, I cannot easily detect this from the new installer.
But there's an easy fix: go to your Windows `Settings`, `Add or remove programs`, `Search this list` for `BrickStore` and then uninstall any version that comes up.
Afterwards, go to https://github.com/rgriebl/brickstore/releases, download the current installer directly and install that.
